lib/serialisers/JSON06Serialiser.js

```diff
@@ -121,8 +121,11 @@ module.exports = class JSON06Serialiser extends JSONSerialiser {
       attributes.set('default', new this.namespace.elements.Array([defaultValue.content]));
     }
 
+    // Strip typeAttributes from samples, 0.6 doesn't usually contain them in samples
     samples.forEach((sample) => {
-      sample.content.attributes.remove('typeAttributes');
+      if (sample.content && sample.content.element) {
```

Comment:
This is the fix for the particular exception, the "content" of the sample (object) is an array in this case and thus does not have "attributes" (which returns undefined). The fix here is adding a "cheap" check if the sample's content exists, and it is an element.